src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  // The car coste calculator
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= LONG_TERM_DISCOUNT;
+
+    return price;
+  }
+
+  if (days >= 3) {
```

Comment:
Those numbers are called magic numbers. Avoid using magic numbers and keep them in variables with descriptive name.